content/en/docs/concepts/storage/volume-pvc-datasource.md

```diff
@@ -36,6 +35,7 @@ Users need to be aware of the following when using this feature:
 * Cloning is only supported within the same Storage Class.
     - Destination volume must be the same storage class as the source
     - Default storage class can be used and storageClassName omitted in the spec
+* Cloning can only be performed between two volumes that use the same VolumeMode setting (if you request a block mode volume, the source MUST also be block mode)
```

Comment:
nit:
```suggestion
* Cloning can only be performed between two volumes that use the same `volumeMode` setting.  
  For example, if you request a block mode volume, the source volume must also be block mode.
```

If you want to call this text out, consider putting it inside a `{{< note >}}` shortcode block.

Reply:
@sftim I think I covered this in an update, let me know if I got everything you were thinking there.